.github/workflows/test.yml

```diff
@@ -4,23 +4,18 @@ on: [push, pull_request]
 
 jobs:
   compilation-tests:
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
+    container: ptisserand/kakarot-builder:latest
```

Comment:
add a Dockerfile in the project for building this image ourselves

Reply:
ok created an issue